include/evhtp/evhtp.h

```diff
@@ -441,7 +442,9 @@ struct evhtp_connection {
     struct timeval    send_timeo;                  /**< conn write timeouts (overrides global) */
     evutil_socket_t   sock;
     evhtp_request_t * request;                     /**< the request currently being processed */
+    uint64_t          max_headers_size;
     uint64_t          max_body_size;
+    uint64_t          bytes_read;                  /**< the size that current request has read */
```

Comment:
This is already in `struct htparser`, the field being `bytes_read`. It is also a uint16.

Reply:
Is bytes_read just the size of each parse? What if the data were deformed?

For example
A malicious client send data: 
**Hossssssssssssssssssssssssssssssssssssssssssssssssss...(very long)**

after a few seconds, the client send data next part:
**sssssssssssst: www.googleeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee....(very long, without '\r\n')**

Is it possible to overflow the memory?